projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -407,3 +408,23 @@ export function containsBackupRealtime( planClass ) {
 		'is-realtime-backup-plan',
 	].includes( planClass );
 }
+
+/**
+ * Security Daily/Realtime plan no longer includes VideoPress as of Oct 7 2021 00:00 UTC.
+ * This check identifies purchases of Security Daily/Realtime purchased before or after that date.
+ *
+ * @param {*} purchase - The site purchase object.
+ * @returns {boolean} True if legacy plan (VideoPress is included), false otherwise.
+ */
+export const isVideoPressLegacySecurityPlan = purchase =>
+	purchase.active &&
+	includes(
+		[
+			PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+			PLAN_JETPACK_SECURITY_DAILY,
+			PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+			PLAN_JETPACK_SECURITY_REALTIME,
+		],
+		purchase.product_slug
+	) &&
+	moment( purchase.subscribed_date ).isBefore( moment.utc( '2021-10-07' ) );
```

Comment:
I wonder if we could bypass the new dependency and rely on something like `isInTheFuture` from `@wordpress/date`?

Reply:
I don't think so. `isInTheFuture` only takes 1 parameter so I can't set a starting date for the comparison. The rest of the date module doesn't seem to add much functionality to do this type of check either.

The dependency itself is imported by WordPress, it looks like `@wordpress/date` uses momentjs internally, but I guess there is risk if that ever changes.

I could eliminate the helpers altogether and replace it with a pure js implementation, since the `subscribed_date` property includes timezone info.
```suggestion
	new Date( purchase.subscribed_date ) < new Date( '2021-10-07T00:00:00+00:00' );
```

Reply:
I've made the change, and reset all the webpack/package.json changes as well.

This makes things much simpler, and eliminates the wpcom diff too!